src/main/java/cz/cvut/kbss/termit/persistence/dao/VocabularyDao.java

```diff
@@ -223,16 +224,28 @@ public void refreshLastModified(RefreshLastModifiedEvent event) {
     public List<AbstractChangeRecord> getChangesOfContent(Vocabulary vocabulary) {
         Objects.requireNonNull(vocabulary);
         final List<URI> terms = em.createNativeQuery("SELECT DISTINCT ?term WHERE {" +
-                "GRAPH ?vocabulary { " +
-                "?term a ?type ;" +
-                "}" +
-                "?term ?inVocabulary ?vocabulary ." +
-                " }", URI.class).setParameter("type", URI.create(SKOS.CONCEPT))
-                .setParameter("vocabulary", vocabulary).getResultList();
+            "GRAPH ?vocabulary { " +
+            "?term a ?type ;" +
+            "}" +
+            "?term ?inVocabulary ?vocabulary ." +
+            " }", URI.class).setParameter("type", URI.create(SKOS.CONCEPT))
+                                  .setParameter("vocabulary", vocabulary).getResultList();
         return terms.stream().flatMap(tUri -> {
             final Term t = new Term();
```

Comment:
Not a fancy pattern - the findAll method only does t.getUri anyway.

Reply:
I don't understand. Besides, this method has been there for ages, it appears changed only due to the formatting.

Reply:
Yeah, true, just reformatting.

My point was that the key - tUri - seems to be enough to find a list of terms, no need to pass the whole asset. We use this e.g. in VocabularyDao. But we are not consistent through Termit with this.

Reply:
I see. Yes, the Term URI is enough. However, I try to use domain objects in the APIs throughout the application (yes, probably not everywhere), so that is the reason for the pattern here.